### PR TITLE
Apply OpenAPI client code changes for F2

### PR DESCRIPTION
**Pull request by rg911**
- Latest OpenAPI spec for F2 applied
- Fixed Receipt serialization bug 
- Added missing e2e tests for MosaicOwner endpoints

### CODE REVIEW

File: src/infrastructure/MetadataHttp.ts [hunk 1]
```diff
@@ -259,7 +231,7 @@ export class MetadataHttp extends Http implements MetadataRepository {
                 metadataEntry.targetPublicKey,
                 UInt64.fromHex(metadataEntry.scopedMetadataKey),
                 metadataEntry.metadataType.valueOf(),
-                metadataEntry.valueSize,
+                Convert.decodeHex(metadataEntry.value).length,
```
Comment (fboucquez) [inline src/infrastructure/MetadataHttp.ts]: Should we just remove the size from the transaction model? I took it out in Java

Comment (evias) [reply src/infrastructure/MetadataHttp.ts]: what is the reasoning behind removing it ?

Comment (rg911) [reply src/infrastructure/MetadataHttp.ts]: this size is not defined in OpenAPI, and I believe because it is not returning from the rest.

Comment (rg911) [reply src/infrastructure/MetadataHttp.ts]: It can be easily calculated from `value.length` I think. as how the SDK is doing here

**Pull request merged**